server/src/pages/shotindex/view.js

```diff
@@ -47,6 +48,9 @@ class Body extends React.Component {
         <div id="shot-index">
           {children}
         </div>
+        <Footer>
+          <a href="/delete-account">Disable account / delete all shots</a>
```

Comment:
I think Delete account is a better thing to say. Disable sounds like it can be re-enabled.